app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/upcoming/MealBottomSheet.kt

```diff
@@ -0,0 +1,224 @@
+        }
+        IconButton(
+            onClick = {
+                selectedFilters.clear()
+                if (!selectedFilters.contains(Filter.LUNCH)) {
+                    selectedFilters.add(Filter.LUNCH)
+                }
+            },
+            modifier = Modifier
+                .align(CenterVertically)
+                .padding(end = 16.dp, start = 100.dp),
+
+            ) {
+            if (selectedFilters.contains(Filter.LUNCH)) {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_selected),
+                    contentDescription = "Lunch",
+                )
+            } else {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_unselected),
+                    contentDescription = "Lunch",
+                )
+            }
+        }
+
+    }
+    Spacer(
+        modifier = Modifier
+            .padding(start = 12.dp, end = 16.dp, top = 12.dp, bottom = 8.dp)
+            .fillMaxWidth()
+            .height(1.dp)
+            .background(GrayZero, CircleShape)
+    )
+    Row(horizontalArrangement = Arrangement.SpaceBetween, modifier = Modifier.fillMaxWidth()) {
+        Column {
+            Text(
+                text = "Dinner",
+                modifier = Modifier.padding(start = 16.dp),
+                style = EateryBlueTypography.h5
+            )
+            Text(
+                text = "5:00 PM-8:30 PM",
+                modifier = Modifier.padding(start = 16.dp),
+                style = EateryBlueTypography.caption
+            )
+        }
+        IconButton(
+            onClick = {
+                selectedFilters.clear()
+                if (!selectedFilters.contains(Filter.DINNER)) {
+                    selectedFilters.add(Filter.DINNER)
+                }
+            },
+            modifier = Modifier
+                .align(CenterVertically)
+                .padding(end = 16.dp, start = 100.dp),
+
+            ) {
+            if (selectedFilters.contains(Filter.DINNER)) {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_selected),
+                    contentDescription = "Dinner",
+                )
+            } else {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_unselected),
+                    contentDescription = "Dinner",
+                )
+            }
+        }
+    }
+    Button(
+        onClick = { hide() },
+        shape = RoundedCornerShape(32.dp),
+        modifier = Modifier
+            .padding(top = 20.dp, start = 16.dp, end = 16.dp)
+            .fillMaxWidth(),
+        colors = ButtonDefaults.buttonColors(
+            backgroundColor = EateryBlue,
+            contentColor = Color.White
+        )
+    ) {
+        Text(
+            text = "Show Menu",
+            style = EateryBlueTypography.h5,
+            modifier = Modifier.padding(top = 8.dp, bottom = 4.dp)
+        )
+    }
+
+    TextButton(
```

Comment:
As we previously talked about, it seems reset isn't appearing on the screen. 
https://developer.android.com/reference/kotlin/androidx/compose/material/ModalBottomSheetValue#HalfExpanded
If you look at that link, you will see that the state can be hidden, half expanded or fullexpanded. Half expanded takes up half the screen which is why it seems the components don't seem to be showing. So setting the state to fully expanded should fix it.